docker/prod/container-start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+cd /code && \
+webpack
+python manage.py runserver 0.0.0.0:8000
```

Comment:
On production environment, we won't use `runserver` command. Can you please configure it using uwsgi with nginx?